addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1351,6 +1372,225 @@ QUnit.test('chat window: switch on TAB', async function (assert) {
     );
 });
 
+QUnit.test('chat window: scroll position conservation on folded', async function (assert) {
```

Comment:
specify it's for chat window with a thread (keep scroll position in message list)